OpenOversight/tests/routes/test_user_api.py

```diff
@@ -25,36 +25,24 @@
 # All login_required views should redirect if there is no user logged in
 @pytest.mark.parametrize("route,methods", routes_methods)
 def test_user_api_login_required(route, methods, client, mockdata):
-    if HTTPMethod.GET in methods:
-        rv = client.get(route)
-        assert rv.status_code == HTTPStatus.FORBIDDEN
-    if HTTPMethod.POST in methods:
-        rv = client.post(route)
-        assert rv.status_code == HTTPStatus.FORBIDDEN
+    rv = client.get(route)
+    assert rv.status_code == HTTPStatus.FORBIDDEN
```

Comment:
These `if` statements were unnecessary.

Reply:
Is there a reason we're not testing POST routes anymore?

Reply:
Ahhh, I see it now! I'll revert the changes.